static/skin/index.js

```diff
@@ -18,6 +18,20 @@
     let timer;
     let languages = {};
 
+    function updateRssFeedLink() {
+        let tempParams = new URLSearchParams(window.location.search);
+        let keysForDel = [];
+        tempParams.forEach((value, key) => {
+            if (value == '') {
+                keysForDel.push(key);
+            }
+        });
+        keysForDel.forEach(key => {
+            tempParams.delete(key);
+        });
```

Comment:
You can do the same with a single loop if instead of collecting keys to be deleted, you construct a filtered list of params during the first and only iteration:

```js
const inputParams = new URLSearchParams(window.location.search);
const filteredParams = new URLSearchParams();
for (const [key, value] of inputParams) {
    if ( value != '' ) {
        filteredParams.append(key, value);
    }
}
```

Reply:
smart!